src/backend/PortabilityService.Functions/ReportFormat.cs

```diff
@@ -29,20 +28,8 @@ public static class ReportFormat
                 : req.CreateResponse(HttpStatusCode.BadRequest);
         }
 
-        private static readonly ResultFormatInformation[] Formats = new[]
-        {
-            new ResultFormatInformation
-            {
-                DisplayName = "Excel",
-                MimeType = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
-                FileExtension = ".xlsx"
-            },
-            new ResultFormatInformation
-            {
-                DisplayName = "Json",
-                FileExtension = ".json",
-                MimeType = "application/json"
-            }
-        };
+        private static ResultFormatInformation[] Formats { get; } = Report.ReportWriters
```

Comment:
DI?

Reply:
For me, it's more of consistency. I know DI isn't fully supported in functions, but I just want to make sure we're using the appropriate patterns where possible